### DISCUSSION TITLE
Receptor folder not exist error

### DISCUSSION THREAD

**Discussion by zyy6657** [Prospero1988/AutoDock_vina_pipeline#4] [Q&A] [answered]
The error message "Receptor folder '4XHI_B' does not exist" appears when using the "show results" function. I'm not sure what might be causing this issue.
![1](https://github.com/user-attachments/assets/599f5902-89f9-4fd5-b042-48d1063b9dc4)
![3](https://github.com/user-attachments/assets/c88f41d2-b68f-4bed-bff1-11454f0a5fef)
![2](https://github.com/user-attachments/assets/e19ddcd4-3db7-4d15-ba4e-5617f03bbda9)

**Comment by Prospero1988** [comment]
Hi, 
Thank you for your interest in my repo. :-) I am sorry that something is not working as expected. Let's start with the basics, did any folders appear in the local folder on the drive you specified (C:\serslw19\AutoDock_vina_pipeline_results) after starting the calculation? After docking is complete, that's where all the files generated by autodock should appear. If anything appeared there, there should be two log files named docking_error.log and docking_output.log in the directory named after the project you specified . Can you attach them? 

That way I will know where to start. :-)

Project folder should looks like this:

![image](https://github.com/user-attachments/assets/6fcb4933-40cf-445b-a55e-940e94b7fd20)

**Reply by zyy6657** [reply]
Sorry to take up your time. The two files you mentioned are not showing up on my end. As shown in Figure 1, this is the file display inside the container, and Figure 2 shows the files on Windows.
![Fig 1](https://github.com/user-attachments/assets/eaf856f9-3eea-47d0-a293-8e712d7a32b1)
![Fig 2](https://github.com/user-attachments/assets/1fd26d49-2ae8-4efa-818a-e29788adc3c8)

**Reply by Prospero1988** [reply]
Dear @zyy6657

I already know what the problem is. I didn't check this program on the new system. The problem is that when you mount your directory, the files required by the program from the results folder are not there. Everything will work if you copy the contents of the “results” folder from the repository into your local “results” folder that you created. There, there is a template folder and a text file with passwords. The most important thing is this “template” folder.

This is a temporary solution. I will correct this error and upload a new docker image. Thank you for finding this bug.

EDIT:
I've already repaired this bug. By evening I'm going to create docker image and upload it to the Docker Hub :-)

**Reply by Prospero1988** [reply, marked as answer]
Dear @zyy6657

Now everything should be working. Please download a new docker image (from the same link, or with the pull command), mount it  into container with **docker compose up** command and try it out. 
If you have any problems or questions then feel free to discuss. I am very keen on feedback so that the provided repository works and can help someone else with their work.

Have a nice day!

![image](https://github.com/user-attachments/assets/fb6f7f43-7517-49ba-a3ee-29d9b6c0ab8d)
![image](https://github.com/user-attachments/assets/8207ae6d-4b47-44fc-af04-1a3af21ad562)
